packages/stark-ui/src/modules/message-pane/services/message-pane.service.ts

```diff
@@ -67,18 +67,14 @@ export class StarkMessagePaneServiceImpl implements StarkMessagePaneService {
 		if (message instanceof StarkMessageImpl) {
 			normalizedMessage = message;
 		} else {
-			normalizedMessage = new StarkMessageImpl();
-
-			// Only copy the id if it is filled-in, otherwise keep the auto-generated id
-			if (message.id && message.id.length > 0) {
-				normalizedMessage.id = message.id;
-			}
-
-			normalizedMessage.key = message.key;
-			normalizedMessage.interpolateValues = message.interpolateValues;
-			normalizedMessage.code = message.code;
-			normalizedMessage.type = message.type;
-			normalizedMessage.priority = message.priority;
+			normalizedMessage = new StarkMessageImpl(
+				message.id,
```

Comment:
We should check that the `message.id` is not an empty string (""), if that is the case then we should pass `undefined` so the `StarkMessageImpl` can set a default id.

Reply:
The `StarkMessageImpl` should cover that case through `this.id = id || _uniqueId();`

Reply:
Ah yes, my bad, I was thinking that empty string was evaluated to `true` but it is not the case :p